services/familyClassifiers.service.ts

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import moleculer, {Context} from 'moleculer';
+import { Method, Service } from 'moleculer-decorators';
+
+import DbConnection from '../mixins/database.mixin';
+import {
+  COMMON_DEFAULT_SCOPES,
+  COMMON_FIELDS,
+  COMMON_SCOPES,
+  CommonFields,
+  CommonPopulates,
+  RestrictionType,
+  Table,
+  throwValidationError,
+} from '../types';
+import { UserAuthMeta } from "./api.service";
+
+
+interface Fields extends CommonFields {
+  id: number;
+  name: string;
+  nameLatin: string;
+
+}
+
+interface Populates extends CommonPopulates {}
+
+export type FamilyClassifier<
+  P extends keyof Populates = never,
+  F extends keyof (Fields & Populates) = keyof Fields,
+> = Table<Fields, Populates, P, F>;
+
+@Service({
+  name: 'familyClassifiers',
+  mixins: [DbConnection({
+    collection: 'familyClassifiers',
+  })],
+  settings: {
+    fields: {
+      id: {
+        type: 'string',
+        columnType: 'integer',
+        primaryKey: true,
+        secure: true,
+      },
+      name: 'string|required',
+      nameLatin: 'string|required',
```

Comment:
siulyciau dar paruosti virtual field populate'a rusims. not needed bet gal pravers